portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -35,12 +41,14 @@
   private final String NAME = "name";
   private final String MOOD = "mood";
   private final String CONTENT = "content";
+  private final String SENTIMENT = "sentiment";
   private final String TIMESTAMP = "timestamp";
```

Comment:
I think I forgot to mention in my previous review: for constants, declare them as private final static.

Reply:
Got it